icu4c/source/common/rbbirb.cpp

```diff
@@ -68,7 +68,6 @@ RBBIRuleBuilder::RBBIRuleBuilder(const UnicodeString   &rules,
     fLBCMNoChain        = FALSE;
     fLookAheadHardBreak = FALSE;
     fUSetNodes          = NULL;
-    fRuleStatusVals     = NULL;
```

Comment:
Bad change. If something goes wrong before this field is assigned a new UVector, then the object is left with this field unassigned.